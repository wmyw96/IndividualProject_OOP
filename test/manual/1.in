...###....
.#.###3...
5.........
1.........
..........
.1...#####
5..4.#####
..4###....
...#.3....
...#......
