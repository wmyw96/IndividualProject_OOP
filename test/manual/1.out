Maximize the total connected pairs: 2 pairs
...***....
.*.***....
..........
1.........
1.........
11...*****
..44.*****
..4***....
...*......
...*......